File: frontend/check_audio.js
const http = require('http');

const options = {
    hostname: 'localhost',
    port: 3000,
    path: '/sounds/notification.mp3',
    method: 'HEAD'
};

const req = http.request(options, (res) => {
    console.log(`Status Code: ${res.statusCode}`);
    console.log(`Content-Type: ${res.headers['content-type']}`);
    console.log(`Content-Length: ${res.headers['content-length']}`);
});

req.on('error', (e) => {
    console.error(`Problem with request: ${e.message}`);
});

req.end();

File: frontend/check_audio.py
import requests

try:
    response = requests.head('http://localhost:3000/sounds/notification.mp3')
    print(f"Status Code: {response.status_code}")
    print(f"Content-Type: {response.headers.get('Content-Type')}")
    print(f"Content-Length: {response.headers.get('Content-Length')}")
except Exception as e:
    print(f"Error: {e}")
